lib/index.ts

```diff
@@ -215,15 +216,30 @@ async function getManifestFiles(targetImage: string, options?: any) {
     return [];
   }
 
-  let excludeGlobs: string[] = [];
+  const excludeGlobs: string[] = ["/sys/**"];
   if (options.manifestExcludeGlobs) {
-    excludeGlobs = options.manifestExcludeGlobs as string[];
+    excludeGlobs.concat(options.manifestExcludeGlobs as string[]);
   }
 
   const globs = options.manifestGlobs as string[];
   const docker = new Docker(targetImage, options);
 
-  let files = await docker.findGlobs(globs, excludeGlobs);
+  const interFiles = await docker.findGlobs(globs, excludeGlobs, "/", false);
+
+  let files: string[] = flatten(
+    Promise.all(
```

Comment:
Nice one - this should reduce the overhead of running `ls` multiple times!

Reply:
but the test still fails :(